sensor/tag_based_pnp_calibrator/README.md

```diff
@@ -0,0 +1,116 @@
+# tag_based_pnp_calibrator
+
+A tutorial for this calibrator can be found [here](../docs/tutorials/tag_based_pnp_calibrator.md)
+
+## Purpose
+
+The package `tag_based_pnp_calibrator` allows extrinsic calibration among the camera sensor and lidar sensor used in autonomous driving and robotics.
```

Comment:
```suggestion
The package `tag_based_pnp_calibrator` allows extrinsic calibration among the camera and lidar sensors used in autonomous driving and robotics.
```

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/32dd61431d3beb3d84ecb085cd246042de8efb09